File: C/troco_verificado.c
#include <stdio.h>
#include <locale.h>

int quantidade;
double preco, dinheiro, troco;

int main() {
    setlocale(LC_ALL, "Portuguese");

    printf("Preço unitário do produto: ");
    scanf("%lf", &preco);
    printf("Quantidade comprada: ");
    scanf("%d", &quantidade);
    printf("Dinheiro recebido: ");
    scanf("%lf", &dinheiro);

    troco = preco * quantidade - dinheiro;

    if (troco > 0) {
        printf("DINHEIRO INSUFICIENTE. FALTAM %.2lf REAIS", troco);
    }
    else {
        troco = dinheiro - preco * quantidade;
        printf("TROCO = %.2lf", troco);
    }

}

File: Python/troco_verificado.py
preco = float(input("Preço unitário do produto: "))
quantidade = int(input("Quantidade comprada: "))
dinheiro = float(input("Dinheiro recebido: "))

total = preco * quantidade
troco = dinheiro - total

if dinheiro < total:
    print(f"DINHEIRO INSUFICIENTE. FALTAM {troco * -1:.2f} REAIS")
else:
    print(f"TROCO = {troco:.2f}")
